apps/web/src/pages/Pools/AssessmentPlanBuilderPage/utils.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/prefer-default-export */
+import { IntlShape } from "react-intl";
+
+import { AssessmentStep } from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getAssessmentStepType,
+  getLocale,
+} from "@gc-digital-talent/i18n";
+
+export const assessmentStepDisplayName = (
+  assessmentStep: AssessmentStep,
+  intl: IntlShape,
+): string => {
+  const locale = getLocale(intl);
+  // don't want "N/A" from getLocalizedName
+  const localizedTitle = assessmentStep?.title
+    ? assessmentStep.title[locale]
+    : null;
```

Comment:
Not sure if this helps, but `getLocalizedName` has an option to return an empty string instead of "N/A".

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f2fe5fc70faa855b742591b03cfd6657b8d1446b/packages/i18n/src/utils/localize.ts#L65-L74

Reply:
Thanks, I didn't know about that.
[use getLocalizedName](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8403/commits/8389ba2954b41972e340b6b51e84f6b18f53b9a2)